### ISSUE TITLE
[Feature] FAQ Accordion not Accessible

### ISSUE THREAD

**Issue by Queen-codes** [cc-archive/legaldb#199]
## Problem
The FAQ section is not accessible while navigating with keyboard navigation alone. According to accessibility guidelines, all interactive elements should be fully navigable and operable using a keyboard. This is currently not the case.

## Description
The FAQ section should be accessible for all users. The section needs to respond to keyboard inputs (e.g., tabbing through questions and using the enter or space keys to expand/collapse answers). 

## Additional Context
- Link to FAQ section: [LegalDb FAQ Page](https://legaldb.creativecommons.org/faq/)
- Link to additional information: (https://www.w3.org/WAI/ARIA/apg/patterns/accordion/)
- creativecommons/vocabulary/issues/55

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by TimidRobot** [comment]
https://github.com/creativecommons/legaldb/labels/%F0%9F%9A%A7%20status%3A%20blocked by:
- creativecommons/vocabulary/issues/55

@Queen-codes please add a comment to creativecommons/vocabulary/issues/55 about the need for accessibility in any according implementation

**Comment by Queen-codes** [comment]
okay, got it. thank you @TimidRobot
